module-bsp/board/rt1051/bsp/cellular/rt1051_cellular.cpp

```diff
@@ -221,15 +221,20 @@ namespace bsp
             assert(false);
         }
 
-        constexpr auto maxTXCheckRetires = 5;
+        constexpr auto maxTXCheckRetries = 10;
+        int retries;
```

Comment:
```suggestion
        int retries = 0;
```

Reply:
it is set I the for look we don't need to set it to zero here